contribs/gmf/src/datasource/Helper.js

```diff
@@ -3,7 +3,7 @@ import gmfEditingEnumerateAttribute from 'gmf/editing/EnumerateAttribute.js';
 import ngeoDatasourceHelper from 'ngeo/datasource/Helper.js';
 import ngeoFormatAttributeType from 'ngeo/format/AttributeType.js';
 
-class Helper {
+export class Helper {
```

Comment:
Please rename it to `DatasourceHelper` to don't have to rename ii if we import it :-)